### ISSUE TITLE
Brooken google group link in documentation

### ISSUE THREAD

**Issue by DavidDTA** [pholser/junit-quickcheck#411]
The [documentation](https://pholser.github.io/junit-quickcheck/site/1.0/discuss.html) has a [link](https://groups.google.com/d/forum/junit-quickcheck) to a google group, which says "Content unavailable" when followed.

**Comment by sohah** [comment]
@pholser, can someone follow up on that link, please? I want to use junit-QuickCheck in my research, and I wanted to seek ask questions or find out if people have already answered them on previous threads on the google group. It would be great if we have this link working again, failing that, what would be the best way to ask questions and seek help. Thanks!

**Comment by pholser** [comment]
Corrected on master.

https://groups.google.com/u/1/g/junit-quickcheck

**Comment by sohah** [comment]
I get "content unavailable" message when I click on the link above. Perhaps this is related to this issue #146

**Comment by pholser** [comment]
try now?

**Comment by sohah** [comment]
Yes, that now works. I no longer get "Content unavailable" but I am prompted to join the group. Thank you for addressing this!

**Comment by DavidDTA** [comment]
One note on the new url you posted: Google's apps unfortunately hardcode which user you're logged in with into the url.  I suggest removing `/u/1` from the url to make it more versitile in case someone has access to the group via a user which is not second in their user list.

**Comment by pholser** [comment]
Done.
